pkg/kubelet/server/server.go

```diff
@@ -282,7 +284,7 @@ func (s *Server) InstallDefaultHandlers() {
 const pprofBasePath = "/debug/pprof/"
 
 // InstallDeguggingHandlers registers the HTTP request patterns that serve logs or run commands/containers
-func (s *Server) InstallDebuggingHandlers() {
+func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
```

Comment:
This is a workaround for the in-process docker CRI shim to reuse the same 10250 port. We will eventually need to handle serving requests on a different port and decouple the logic here. 
Whatever has the smallest footprint and affects other runtimes the least would be my choice.